home-assistant-addon-proxy/CHANGELOG.md

```
@@ -1,5 +1,10 @@
 # ChangeLog
 
+## 0.7.0
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)